src/components/resources/resources.css

```diff
@@ -2,9 +2,13 @@
   text-align: center;
   margin-top: 2rem;
 }
+
 .resources__container .btn {
   margin: 10px;
 }
 .resources__container p {
   text-align: left;
 }
+span {
```

Comment:
can you change it to `resources__container span` that way it only affects the span tags in the resource page

Reply:
Okay, that also works.